### ISSUE TITLE
[Bug]: [Perf][Upgrade]Search performance degrade a lot and recall become 1.0 from 0.983 after upgrading 2.1.2 to master-20220925-91df8f2d

### ISSUE THREAD

**Issue by zhuwenxing** [milvus-io/milvus#19429]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Environment

```markdown
- Milvus version: v2.1.2 --> master-20220925-91df8f2d
- Deployment mode(standalone or cluster): standalone and cluster 
- SDK version(e.g. pymilvus v2.0.0rc2):
- OS(Ubuntu or CentOS): 
- CPU/Memory: 
- GPU: 
- Others:
```

### Current Behavior

First recall test
```
[2022-09-25T09:10:14.090Z] Search...
[2022-09-25T09:10:14.090Z] search cost  4.7969 seconds
[2022-09-25T09:10:14.090Z] recall=0.983
```

Second recall test
```
[2022-09-25T09:19:34.242Z] Search...
[2022-09-25T09:19:34.242Z] search cost  22.4107 seconds
[2022-09-25T09:19:34.242Z] recall=1.0
```

### Expected Behavior

Take v2.1.2 --> 2.1.2-latest as example

https://qa-jenkins.milvus.io/blue/organizations/jenkins/deploy_test_for_release/detail/deploy_test_for_release/194/pipeline

first recall test
```
[2022-09-24T09:16:46.328Z] Search...
[2022-09-24T09:16:46.328Z] search cost  4.8564 seconds
[2022-09-24T09:16:46.328Z] recall=0.983
```

second recall test
```
[2022-09-24T09:27:44.284Z] Search...
[2022-09-24T09:27:44.284Z] search cost  4.1990 seconds
[2022-09-24T09:27:44.284Z] recall=0.983
```

### Milvus Log

failed job: https://qa-jenkins.milvus.io/blue/organizations/jenkins/deploy_test/detail/deploy_test/297/pipeline/296
collection name: `sift_128_euclidean`

log:

[artifacts-standalone-upgrade-297-server-logs.tar.gz](https://github.com/milvus-io/milvus/files/9642591/artifacts-standalone-upgrade-297-server-logs.tar.gz)

[artifacts-standalone-upgrade-297-pytest-logs.tar.gz](https://github.com/milvus-io/milvus/files/9642592/artifacts-standalone-upgrade-297-pytest-logs.tar.gz)

### Anything else?

In the second recall test, the recall is 1.0 and the search time increases a lot, is it because the search becomes a brute force search?

**Comment by yanliang567** [comment]
no indexed was issued when searching
```
[2022/09/25 09:22:08.680 +00:00] [DEBUG] [querynode/segment.go:356] ["do search on segment done"] [msgID=436233187442556994] [segmentID=436233079839522818] [segmentType=Growing] [loadIndex=false]
[2022/09/25 09:22:08.681 +00:00] [DEBUG] [querynode/segment.go:356] ["do search on segment done"] [msgID=436233187442556994] [segmentID=436233081123766273] [segmentType=Growing] [loadIndex=false]
[2022/09/25 09:22:08.689 +00:00] [DEBUG] [querynode/segment.go:356] ["do search on segment done"] [msgID=436233187442556994] [segmentID=436233084164636674] [segmentType=Growing] [loadIndex=false]

```

**Comment by yah01** [comment]
/assign
/assign @zhuwenxing 
plz retry with the latest commit, the PR #19414 should fix this

**Comment by longjiquan** [comment]
This issue was already fixed by #19709 , please help to recheck this, @zhuwenxing , thx.
/assign @zhuwenxing
